### DISCUSSION TITLE
[Question] How can i use different sites or mirrors for some binary packages

### DISCUSSION THREAD

**Discussion by chadi-kazan** [yarnpkg/berry#4819] [Q&A] [answered]
I am migrating from npm to yarn berry. my company blocks installation of some packages with native bindings like node-sass and canvas, so what we do is download them and host them on an internal mirror and set variables in .npmrc 

something like this
```json
sass_binary_site=http://internalurl.company.com/repos/node-sass/raw/releases/download
puppeteer_skip_chromium_download=true
canvas_binary_host_mirror=http://internalurl.company.com/repos/mirrors/raw/node-canvas/releases/download
```
I obviously can no longer set those in .yarnrc.yml and it doesn't pick them up if i define them as env variables ( or maybe i am doing that wrong?)
can anyone help please?
thanks

**Comment by arcanis** [comment, marked as answer]
Individual packages can't be fetched from specific registries. We support custom registries on a by-scope basis, but not by-package.

**Reply by chadi-kazan** [reply]
so what if a package is not scoped, like node-sass for instance? yarn ignores those?
